tests/components/deconz/test_sensor.py

```diff
@@ -155,6 +141,10 @@ async def test_sensors(hass):
     }
     gateway.api.event_handler(state_changed_event)
 
+    assert hass.states.get("sensor.light_level_sensor").state == "1.6"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)